src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -52,6 +54,13 @@ export const Details: FC<DetailsProps> = ({ conversation, ...props }) => {
   const item =
     conversation.items?.[0]?.item?.__typename !== "%other" &&
     conversation.items?.[0]?.item
+
+  const attachments = conversation.messagesConnection.edges
+    .map(({ node }) => node.attachments)
+    .filter(attachments => attachments.length > 0)
+    .reduce((previous, current) => previous.concat(current), [])
+    .filter(attachment => !attachment.contentType.includes("image"))
```

Comment:
Eventually it'd be nice to replace this (and the related messagesConnection query) to be pulled directly from the conversation. As is, attachments in the details view will only render if they've already been loaded w/ the messages